contracts/contracts/lib/LibStaking.sol

```diff
@@ -506,9 +506,6 @@ library LibStaking {
         s.validatorSet.recordWithdraw(validator, amount);
         // confirm deposit that updates the confirmed collateral
         s.validatorSet.confirmWithdraw(validator, amount);
-
-        // release stake from gateway and transfer to user
-        payable(validator).sendValue(amount);
```

Comment:
I think we can leave `LibStaking` to just bookkeeping. Transfer outside might be better because even though we confirmed it, we might not transfer immediately.

Reply:
resolve first as it's duplicated below